susemanager-utils/susemanager-sls/salt/bootstrap/bootstrap.repo

```diff
@@ -11,6 +11,7 @@ gpgcheck=0
 enabled=1
 autorefresh=1
 keeppackages=0
+priority=1
```

Comment:
This is dangerous. I am not sure if we remove the repo after the bootstrap is done. If that does not work we never get packages from the normal channels installed. If possible remove any kind of repo priority.

Reply:
Removed.